DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development
+
+This doc explains how to set up a development environment, so you can get started
+contributing to `oam-kubernetes-runtime` or build a PoC (Proof of Concept). 
+
+## Prerequisites
+
+1. Golang version 1.12+
+2. Kubernetes version v1.15+ with kubectl configured
+
+## Build
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and clone this project
```

Comment:
```suggestion
* Clone this project
```

I think fork is required only if one guy want to contribute his code. We could write another doc or leave a github workflow doc link here.